src/App.js

```diff
@@ -0,0 +1,38 @@
+import "./App.css";
+import AboutScreen from "./screens/AboutScreen";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import HomeScreen from "./screens/HomeScreen";
+import Container from "react-bootstrap/Container";
+import Navbar from "react-bootstrap/Navbar";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const App = () => {
+  return (
+    <div className="App">
+      <header>
+        <Navbar bg="dark" variant="dark">
+          <Container>
+            <Navbar.Brand href="/">
+              <b>d.e.c.k.y</b> Inspiration Board
+            </Navbar.Brand>
+            <Navbar.Collapse className="justify-content-end">
+              <Navbar.Text>
+                <a href="/about">About </a>
+              </Navbar.Text>
+            </Navbar.Collapse>
+          </Container>
+        </Navbar>
```

Comment:
If this doesn't need to be in the same file as the router to work, I would consider moving this `Navbar` code out to its own component to keep the App file pretty light on setting up UI.